app/src/main/res/layout/fragment_homepage.xml

```diff
@@ -13,17 +13,209 @@
+                android:textSize="17sp"
+                android:translationX="10dp"
+                android:translationY="-270dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
+            <ImageView
+                android:id="@+id/Minus"
+                android:layout_width="70dp"
+                android:layout_height="70dp"
+                android:src="@drawable/minus"
+                android:translationX="30dp"
+                android:translationY="-265dp" />
+
+            <TextView
+                android:id="@+id/Minus_info"
+                android:layout_width="246dp"
+                android:layout_height="65dp"
+                android:gravity="left"
+                android:text="Use the minus icon to signify the decreasing numerical value associated with the statement."
+                android:textColor="@color/white"
+                android:textSize="17sp"
+                android:translationX="150dp"
+                android:translationY="-335dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
             <Button
-                android:id="@+id/button2"
-                android:layout_width="215dp"
+                android:id="@+id/next_info_button"
+                android:layout_width="106dp"
                 android:layout_height="wrap_content"
-                android:text="Button" />
+                android:layout_marginTop="36dp"
+                android:backgroundTint="#3280B4"
+                android:text="@string/next"
+                android:translationX="265dp"
+                android:translationY="-350dp"
+                app:cornerRadius="100dp" />
+
+            <TextView
+                android:id="@+id/Next_info"
+                android:layout_width="244dp"
+                android:layout_height="71dp"
+                android:gravity="left"
+                android:text="The Next button will transport you to the next page, leading to a page titled End Game."
+                android:textColor="@color/white"
+                android:textSize="17sp"
+                android:translationX="10dp"
+                android:translationY="-400dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
+            <Button
+                android:id="@+id/previous_info_button"
+                android:layout_width="106dp"
+                android:layout_height="40dp"
+                android:layout_marginTop="36dp"
+                android:backgroundTint="#3280B4"
+                android:text="@string/previous"
+                android:translationX="10dp"
+                android:translationY="-430dp"
+                app:cornerRadius="100dp" />
+
+            <TextView
+                android:id="@+id/Previous_info"
+                android:layout_width="244dp"
+                android:layout_height="54dp"
+                android:gravity="left"
+                android:text="The Previous button will take you back to the Tele Op page."
```

Comment:
With the way that the text is currently with the Previous_info and Next_info I believe that there is somewhat of a lack of clarity, that may be better with visuals but I can't see them so if you think that the visuals make everything make more sense then you can leave it as is, but this says that moving back will take the user to the teleop page, but pressing the next button moves them to the end game page, but to my knowledge isn't the use already on the endgame page?